packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -377,15 +377,15 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
    * @returns
    */
   prepareDiagnosticEvent(eventData: Event['event'], options: any) {
-    const {meetingId} = options;
+    const {meetingId, triggeredTime} = options;
```

Comment:
do you know if there is any good reason why options are `any` and not properly typed?

Reply:
I don't know but the options are typed in prepareClientEvent and submitMQE before getting passed through to this private function. I'm going to leave it as